### PR TITLE
change deny ttl to 5 sec

### PR DESCRIPTION
**Pull request by mandarjog**
This PR addresses #1373 .

1. Make all denier options configurable :-) 
2. Change default ttl  to `5s` from `1000 s`

**Release note**:

```release-note
NONE
```

---
This change is [Reviewable](https://reviewable.io/reviews/istio/mixer/1378)

### CODE REVIEW

File: adapter/denier/denier.go [hunk 1]
```diff
@@ -39,15 +39,15 @@ type handler struct {
 func (h *handler) HandleCheckNothing(context.Context, *checknothing.Instance) (adapter.CheckResult, error) {
 	return adapter.CheckResult{
 		Status:        h.status,
-		ValidDuration: 1000 * time.Second,
+		ValidDuration: 5 * time.Second,
```
Comment (douglas-reid) [inline adapter/denier/denier.go]: Should these values be constants, annotated with comments on their setup?  Or maybe even config params for denier handlers?

**Comment by douglas-reid** [comment]
In general, this seems fine to me.

I wonder, however, if we can ever decide universally what the value should be. This feels like it should have a sensible default (and maybe that is: `5s` or `1000` uses), but be configurable *per handler* (so that I can have some deny rules that I don't mind caching for 10 minutes and some that I want to be very responsive).

**Comment by mandarjog** [comment]
Denier `now with more config`  :-)  PTAL

**Comment by mandarjog** [comment]
@myidpt this addresses the cache time issue.

**Comment by myidpt** [comment]
Great fix, Mandar 👍

**Pull request merged**